contracts/extbridge/ext_bridge.sol

```diff
@@ -77,12 +113,12 @@ contract ExtBridge is BridgeTransferERC20, BridgeTransferERC721 {
         uint64 _requestNonce,
         uint64 _requestBlockNumber,
         string memory _tokenURI,
-        uint256[] memory _extraData
+        bytes memory _extraData
     )
         public
     {
         if (_extraData.length > 0) {
-            uint256 offerPrice = _extraData[0];
+            uint256 offerPrice = abi.decode(_extraData, (uint256));
```

Comment:
In my test, the decoder reverted if the size of calldata is less than 32. 
```
The ABI decoder reverts in the beginning of functions and in abi.decode() if passed calldata is too short or points out of bounds. Note that dirty higher order bits are still simply ignored.
```
Refer: https://solidity.readthedocs.io/en/v0.5.11/050-breaking-changes.html?highlight=abi.decode

Reply:
Good. :)
So you suggested to change the length check like below? :)
```
f (_extraData.length >= 32) {
```
How about remote the length check? :)

Or Are you thinking that this way doesn't have any benefit for the size?

Reply:
How about explicitly checking the length if `price` is mandatory? 
`require(_extraData.length == 32, "the price should be uint256")`

Reply:
How about remove and if encode is fail, the tx revert? :)

Reply:
I think in this example, the price is not mandatory. So Could I keep this code? :)

Reply:
I reflected your review. :) Thanks.